lib/psql.js

```diff
@@ -51,6 +52,7 @@ function * exec (db, query) {
   let configs = bastion.getConfigs(db)
 
   yield bastion.sshTunnel(db, configs.dbTunnelConfig)
+  debug('Running query: %s', query.trim())
```

Comment:
moved it down, my original thought was that execPsql already looked a bit overloaded and could maybe be broken up into a couple functions, but on closer look it seems fine